addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRule.java

```diff
@@ -719,14 +737,20 @@ private static String insertUninitVar(String cmd) {
     }
 
     private AlertBuilder buildAlert(
-            String param, String attack, String evidence, String otherInfo, HttpMessage msg) {
+            String param,
+            String attack,
+            String evidence,
+            String otherInfo,
+            TestType type,
```

Comment:
Since the type is now a parameter the other info should be handled inside as well for consistency.